content/docs/user-guide/how-to/add-output-to-stage.md

```diff
@@ -0,0 +1,47 @@
+# Add Output to Stage
+
+There are situations where we execute a stage without specifying all the
+<abbr>outputs</abbr> in `dvc.yaml` (either via `dvc run` or manually), and then
+later want to add more outputs to that stage.
+
+Re-executing a stage using `dvc run -f` to add an output is an expensive step,
+as it requires time and resources. Follow the steps below to add an output to
+the stage without executing it again.
```

Comment:
Still needs to be a little more general like we discussed. And no need to mention dvc run -f here.